### DISCUSSION TITLE
WiFi disappeared on the OTGW?

### DISCUSSION THREAD

**Discussion by wimwis** [rvdbreemen/OTGW-firmware#236] [Q&A]
I bought the WiFi OpenTherm Gateway from Nodo Shop in Holland with an already programmed WeMos D1 Mini (ESP8266).
Its WiFi was briefly available under the name: OTGW-BC-DD-C2-88-84-56, but soon disappeared.
According to Robert van den Breemen's Wiki, the OTGW would enter AP config mode after a few seconds, and the blue LED would remain on constantly.
For me it is only 1 LED, and it keeps flashing.
But via the IP address I did reach the built-in menu with the title “OTGW firmware”.
1.
Is that what Ronald van den Breemen calls: the web UI?
2.
And does that also mean that there is actually nothing wrong?

And if nothing is wrong, then a third question:
3.
I cannot yet connect to the network OTGW-BC-DD-C2-88-84-56 **via WiFi**
(I can only do something via the IP address, as mentioned above.)
And there is no password yet.
What can I do about it?

**Comment by rvdbreemen** [comment]
Hi @wimwis 
Sorry I did not anwser before. I missed it completly.

To answer your questions:
1. Yes, the webui is reached with the browser. 
2. It works as designed, the UI you reach is acutally the "wifi manager" you have to setup your wifi.
3. You need to select your wifi from the UI, and then enter the password of your wifi. This way you setup the ESP so it will automatically logon to your wifi network.

In the wiki you can find this all described and more. Just start here:
https://github.com/rvdbreemen/OTGW-firmware/wiki/Configure-WiFi-settings-using-the-Web-UI

Hope this helps you to get setup,
Robert
